Jenkinsfile

```diff
@@ -8,7 +8,7 @@ kind: Pod
 spec:
   containers:
   - name: maven
-    image: repairnator/ci-env:latest
+    image: andre15silva/repairnator-ci:latest
```

Comment:
added you to https://hub.docker.com/orgs/repairnator such that you can push a new version to the organization instead.

Reply:
Ok, thanks! Will do that.